test/nieSyntaxSpecs.js

```diff
@@ -0,0 +1,42 @@
+describe('Testing nie syntax services', function() {
+	var $injector;
```

Comment:
You can do:

var $injector = angular.injector(['ng','app.services']);
syntaxServices = $injector.get('syntaxServices');

and remove the variable declaration.